.github/workflows/build_app_image.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build App Image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build_and_ship:
+    name: Build & Ship OCI image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # References:
+      # - https://github.com/mamezou-tech/buildpacks-action
+      # - https://github.com/marketplace/actions/buildpacks-action
+      - name: Invoke buildpacks-action
+        uses: mamezou-tech/buildpacks-action@master
+        with:
+          image: 'ghcr.io/hwakabh/bennu-official/django'
+          tag: 'latest'
+          builder: 'paketobuildpacks/builder-jammy-full'
+          # TODO: should be fetched from .python-version
+          env: 'BP_CPYTHON_VERSION=3.11.5'
+          publish: true
```

Comment:
question: Should we need to pass GitHub PAT for pack commands to push image?

Reply:
In [the initial triggered job](https://github.com/hwakabh/bennu-official/actions/runs/7283473776/job/19847404165), we got the following errors, so need to fix this first.

```bash
===> ANALYZING
[analyzer] ERROR: failed to initialize analyzer: validating registry write access: ensure registry read/write access to ghcr.io/hwakabh/bennu-official/django:latest
ERROR: failed to build: executing lifecycle. This may be the result of using an untrusted builder: failed with status code: 1
```

Reply:
Reproduced the issue with local environment. As `pack` command required to `docker login` before build with `--publish` options, so we have to update with additional steps to gh-action config with login to GHCR with GitHub PAT.
```bash
# Validate without login to GHCR
% docker logout ghcr.io     
Removing login credentials for ghcr.io
% cat $HOME/.docker/config.json 
{
	"auths": {
		"https://index.docker.io/v1/": {}
	},
	"credsStore": "osxkeychain",
	"currentContext": "orbstack"
}

% pack build ghcr.io/hwakabh/bennu-official/django:latest --builder paketobuildpacks/builder-jammy-full --env BP_CPYTHON_VERSION=$(cat .python-version) --clear-cache --publish 
latest: Pulling from paketobuildpacks/builder-jammy-full
Digest: sha256:a017ce9f420ac819cce89d3fe2d16a4fa4787601e076c4cd9b8b708e5ef1fe97
Status: Image is up to date for paketobuildpacks/builder-jammy-full:latest
===> ANALYZING
ERROR: failed to initialize analyzer: validating registry write access: ensure registry read/write access to ghcr.io/hwakabh/bennu-official/django:latest
ERROR: failed to build: executing lifecycle: failed with status code: 1
```

Reply:
This would be fixed in #47